packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -446,7 +458,7 @@ function _processSpokeResponse(botKey, spokeResponse, validOutputs, errorOutputs
     errorOutputs[botKey] = {
       status: spokeResponse.status,
       execResponse: spokeResponse.value && spokeResponse.value.execResponse,
-      reason: "empty stdout",
+      message: "empty stdout",
```

Comment:
these errors were lost as they should be described as a `message`, not a `reason`.

note that the previous implementation would have worked fine as well due to this: https://github.com/UMAprotocol/protocol/pull/3699/files#r768479146 change but `message` is more accurate.